server/views/applications/pages/bail-hearing-information/bail-hearing-date.njk

```diff
@@ -0,0 +1,26 @@
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "../../../components/formFields/form-page-date-input/macro.njk" import formPageDateInput %}
+
+{% extends "../layout.njk" %}
+
+{% block questions %}
+
+  <h1 class="govuk-heading-l govuk-!-margin-bottom-4">{{ page.title }}</h1>
```

Comment:
To avoid the extra title, would these options work as you've added on the other pages?

```
legend: {
    text: page.questions.bailHearingDate.question,
    isPageHeading: true,
    classes: "govuk-fieldset__legend--l"
 }
```

Reply:
oh lol, good spot, thanks! not sure how i missed that 🤦